KioskTests/Bid Fulfillment/SwipeCreditCardViewControllerTests.swift

```diff
@@ -5,19 +5,19 @@ import Keys
 import Kiosk
 
 class StubKeys: EidolonKeys {
-    override func cardflightProductionAPIClientKey() -> String! {
+    override var cardflightProductionAPIClientKey: String {
```

Comment:
Yup, the update was in #640: https://github.com/artsy/eidolon/pull/640/files#diff-e79a60dc6b85309ae70a6ea8261eaf95L48 That PR missed the unit testing target, which this PR addresses.